lib/common/containers/App.js

```diff
@@ -212,12 +123,27 @@ export default class App extends React.Component<AppProps> {
       },
       {path: '*', component: loginOptional(PageNotFound)}
     ]
-    const router = (
-      <Router
-        history={browserHistory}
-        onUpdate={logPageView}>
-        {routes.map((r, i) => (<Route {...r} key={i} />))}
-      </Router>
+    const routerWithAuth0 = (
+      <Auth0Provider
+        audience=''
+        // Continue to cache tokens in localstorage (speeds up updating login state when refreshing pages).
+        cacheLocation='localstorage'
+        clientId={AUTH0_CLIENT_ID}
+        domain={AUTH0_DOMAIN}
+        onAccessTokenError={this.handleAccessTokenError}
+        onRedirectCallback={this.handleRedirect}
+        // FIXME: This page will briefly flash after login.
```

Comment:
I haven't been able to log in yet but based on what you said this can probably go!